masking/mask_terms.py

```diff
@@ -127,38 +215,33 @@ def main():
             line = line.rstrip('\n')
 
         if args.unmask:
-            tokens = line.split('\t')
-            if len(tokens) != 3:
-                raise Exception('Need three fields (output, orig source, and masked source)!')
-
-            # output, orig_source, masked_source
-            unmasked = masker.unmask(*tokens)
-            if args.json:
-                jobj['unmasked'] = unmasked
-                print(json.dumps(jobj), flush=True)
-            else:
-                print(unmasked, flush=True)
+            if not args.json:
+                raise Exception('Unmasking requires json format')
+            
+            output = jobj['masked_output']
+            masks = jobj['masks']
+            unmasked = masker.unmask(output, masks)
+            jobj['unmasked'] = unmasked
+            print(json.dumps(jobj), flush=True)
```

Comment:
please add `ensure_ascii=False` to the `dumps` call (otherwise UTF-8 gets formatted in a really ugly way).